RocketChat/Rocket.Chat.ReactNative
# [FIX] Hardcoded backdrop opacity on loading component

## Proposed changes

Apply backdrop opacity theme on Loading component. In addition, added a quick animation to show the backdrop opacity.

## Issue(s)	

Backdrop opacity was fixed for the Light and Dark theme

## How to test or reproduce

The best way to test this task is
- Go to a channel
- Write a message
- Get the `permalink` from that message
- Paste it and send
- Then, when click on the link will appear the Loading

## Screenshots
<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</thead>
</tr>
<body>
<tr>
<td>
<img src="https://user-images.githubusercontent.com/47038980/124662748-99f33d80-de7f-11eb-803e-00655a74e025.png" height="450" />
</td>
<td>
<img src="https://user-images.githubusercontent.com/47038980/124662804-a2e40f00-de7f-11eb-9669-bb09bf27b076.png"  height="450"/>
</td>
</tr>
</tbody>
</table>


## Types of changes




## Checklist



## Further comments
